paybutton/dev/demo/index.html

```diff
@@ -38,7 +38,7 @@
       currency="BCH" text="Purchase" on-transaction="myTransactionFunction"></div>
 
     <div class="paybutton" to="ecash:qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7quss2vxek" goal-amount="2500.05" currency="USD"
-      text="Donate (other dev)" on-success="mySuccessFunction" on-transaction="myTransactionFunction"></div>
+      text="Donate (other dev)" on-success="mySuccessFunction" on-transaction="myTransactionFunction" enable-altpayment="false"></div>
```

Comment:
Can you rebase to the branch hat hasn't been merged yet?

We should have at least 1 contributionOffset example here.